test/models/user_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+
+describe User do
+  # it "does a thing" do
+  #   value(1+1).must_equal 2
+  # end
+  let (:new_user) {
+    User.new(username: "Testing username")
+  }
+
+  describe "relations" do
+    before do
+      new_user.save
+      # second_user = users(:tram)
+
+      @new_work = works(:cowboy)
+      @second_work = works(:undocumented)
+
+      vote_1 = Vote.create(user_id: new_user.id, work_id: @new_work.id)
+      vote_2 = Vote.create(user_id: new_user.id, work_id: @second_work.id)
```

Comment:
When creating data in `before` blocks I recommend using `create!` instead of `create` because it will raise an exception if something goes wrong instead of silently failing to save.